development/circular-deps.jsonc

```diff
@@ -14,7 +14,8 @@
     "ui/pages/confirmations/confirmation/templates/error.js",
     "ui/pages/confirmations/confirmation/templates/index.js",
     "ui/selectors/selectors.js",
-    "ui/store/actions.ts"
+    "ui/store/actions.ts",
+    "ui/store/store.ts"
```

Comment:
@davidmurdoch @itsyoboieltr @MajorLift should we allow this change?

I guess it doesn't add a new circular dependency, but it makes an existing one longer.

Reply:
Ah, it changed since I wrote that comment, the removal is new.